### ISSUE TITLE
Incorrect airport names

### ISSUE THREAD

**Issue by Worf2340** [avwx-rest/AVWX-Engine#16]
KSFO = "International Airport", so does KLAX. They look to be defined wrong in stations.json.

**Comment by flyinactor91** [comment]
I'm currently looking for a new downloadable source for the base dataset. Also working on a script to pull data from APOA for that info and runways, but that will only work for airports in the US.

And the data can be cleaned up in general. The airport elevation is in meters, but the runway lengths are in feet.

**Comment by eagleDiego** [comment]
What was the original source? I also noticed that some newer airports are not included, like `LTFM`.

**Comment by flyinactor91** [comment]
I believe I grabbed it from a GeoNames server in 2015/16 but the link has been lost to me

**Comment by TechnicallyGio** [comment]
Same with Newark

**Comment by flyinactor91** [comment]
The station list has been updated with d31ca0e using http://ourairports.com/data/ as the airport and runway data source. There are some new keys minor changes. For example:

```
{
    "city": "San Francisco",
    "country": "US",
    "elevation_ft": 13,
    "elevation_m": 4,
    "iata": "SFO",
    "icao": "KSFO",
    "latitude": 37.61899948120117,
    "longitude": -122.375,
    "name": "San Francisco International Airport",
    "note": "QSF, QBA",
    "runways": [
        {
            "ident1": "10L",
            "ident2": "28R",
            "length_ft": 11870,
            "width_ft": 200
        },
        {
            "ident1": "10R",
            "ident2": "28L",
            "length_ft": 10602,
            "width_ft": 200
        },
        {
            "ident1": "01R",
            "ident2": "19L",
            "length_ft": 8648,
            "width_ft": 200
        },
        {
            "ident1": "01L",
            "ident2": "19R",
            "length_ft": 7500,
            "width_ft": 200
        }
    ],
    "state": "CA",
    "type": "large_airport",
    "website": "http://www.flysfo.com/",
    "wiki": "http://en.wikipedia.org/wiki/San_Francisco_International_Airport"
}
```

This is currently part of `1.1b2` pending an update to the documentation.

**Comment by flyinactor91** [comment]
Released in v1.1
